projects/1.16.1/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -0,0 +1,452 @@
+	"ars_nouveau.page.amplify_arrow": "在抄入法术的末尾增加两个强化。每次合成产出 32 支箭。",
+	"ars_nouveau.page.pierce_arrow": "在符号链开头添加两个穿刺。每次合成产出 32 支箭。",
+	"ars_nouveau.page.split_arrow": "在法术链开头添加两个分裂。如果法术箭会造成伤害，所有分裂的箭也会。每次合成产出 32 支箭。",
+	"ars_nouveau.bow.invalid": "弓只接受不含方式的法术。",
+	"ars_nouveau.apparatus.norecipe": "找不到配方。",
+	"ars_nouveau.apparatus.nomana": "附近魔力不足。",
+	"ars_nouveau.reset.cleared": "进度已清空。",
+	"effect.ars_nouveau.summoning_sickness": "召唤失调",
+	"ars_nouveau.glyph_desc.accelerate": "增加投射物法术速度。",
+	"ars_nouveau.glyph_desc.amplify": "可叠加地增加大多数法术效果的强度。可增加破坏的挖掘等级和增加法术的伤害。",
+	"ars_nouveau.glyph_desc.aoe": "法术会影响目标方块附近一个更大的范围。",
+	"ars_nouveau.glyph_desc.aquatic": "造成潮涌能量状态。",
+	"ars_nouveau.glyph_desc.blink": "传送施法者。如果击中了实体且施法者副手正持有折越卷轴，则被击中的实体会折越到卷轴上的位置。当对自身使用，施法者会向前闪现。法术炮塔和符文可根据相邻容器内的折越卷轴传送实体。",
+	"ars_nouveau.glyph_desc.break": "你的初始法术。可破坏平均硬度的方块。可被强化以增加挖掘等级。",
+	"ars_nouveau.glyph_desc.cold_snap": "定住已经潮湿或在缓慢状态的实体，并对其造成爆发伤害。附近不在缓慢状态的敌人也会进入缓慢状态。目标附近已经潮湿或缓慢的实体也会被寒流击中。可被强化、延长时效和范围效果增强。",
+	"ars_nouveau.glyph_desc.conjure_water": "在目标位置放置水。可被范围效果增强。",
+	"ars_nouveau.glyph_desc.craft": "打开合成菜单。",
+	"ars_nouveau.glyph_desc.crush": "将石头变成沙砾，沙砾变成沙子。也会伤害实体，会对正在游泳的实体造成额外伤害。",
+	"ars_nouveau.glyph_desc.cut": "剪实体或方块，或者对不能剪的实体造成少量伤害。没有额外消耗。",
+	"ars_nouveau.glyph_desc.dampen": "减少大多数法术的强度。稍微减少魔力消耗，但不会低于 0。",
+	"ars_nouveau.glyph_desc.delay": "放在法术链右边可稍微延迟效果的发生。延迟可被延长时效增强。",
+	"ars_nouveau.glyph_desc.dispel": "移除目标身上的任何药水效果。当对半血以下的女巫使用时，女巫会消失并留下薇克精碎片。也可使已驯服的雏龙、卡班克尔和希尔芙变回护符。",
+	"ars_nouveau.glyph_desc.duration_down": "减少法术持续时间，可影响药水效果、延迟、红石信号等等。",
+	"ars_nouveau.glyph_desc.ender_inventory": "从任何地方打开你的个人末影箱。",
+	"ars_nouveau.glyph_desc.exchange": "对方块使用时会交换该方块和玩家快捷栏中的方块，如同用精准采集挖掘的一样。可用范围效果增强。与高硬度方块交换需要强化。对实体使用则会交换施法者和被击中实体的位置。",
+	"ars_nouveau.glyph_desc.explosion": "在目标位置造成爆炸。",
+	"ars_nouveau.glyph_desc.extend_time": "延长法术持续时间，包括状态效果、唤魔者尖牙和召唤物",
+	"ars_nouveau.glyph_desc.extract": "给破坏添加精准采集效果，并且让爆炸不破坏已掉落的方块。不能与时运叠加。",
+	"ars_nouveau.glyph_desc.fangs": "在法术指向的方向召唤唤魔者尖牙。",
+	"ars_nouveau.glyph_desc.fell": "收获整颗树、蘑菇、仙人掌和其他植物。强化后可破坏更高硬度的材料。范围效果会增加一次能破坏的方块数量。",
+	"ars_nouveau.glyph_desc.flare": "当对着火的实体使用，爆燃会造成爆发伤害并向附近实体蔓延火焰、造成伤害。比伤害符号造成显著更多的伤害。可用延长时效、强化和范围效果增强。",
+	"ars_nouveau.glyph_desc.fortune": "增加用伤害杀死生物和用破坏挖掘方块后的掉落率。不能与提取叠加。",
+	"ars_nouveau.glyph_desc.freeze": "冻结水体，或短时间使目标缓慢。",
+	"ars_nouveau.glyph_desc.gravity": "使方块或实体掉落。",
+	"ars_nouveau.glyph_desc.grow": "使植物生长加速，但这并不会为附近魔力凝集器提供魔力。",
+	"ars_nouveau.glyph_desc.gust": "使目标被推离施法者少量距离",
+	"ars_nouveau.glyph_desc.harm": "你的初始法术。伤害一个目标。可用强化增加伤害，或使用延长时效造成中毒状态。注意，没有延迟的多个伤害会因为击中后的无敌时间无法生效。",
+	"ars_nouveau.glyph_desc.harvest": "当用于成熟的作物，该法术会在不破坏植物的情况下获得完全成熟后的产品。",
+	"ars_nouveau.glyph_desc.haste": "给目标造成速度增益",
+	"ars_nouveau.glyph_desc.heal": "对目标治疗少量生命。当与延长时效一起使用则会变成再生状态，最高 5 级。",
+	"ars_nouveau.glyph_desc.ignite": "使方块或生物短时间着火",
+	"ars_nouveau.glyph_desc.intangible": "使方块暂时变成空气。可用强化增强以处理高挖掘等级方块。还能被范围效果、减少时效和延长时效增强。",
```

```suggestion
	"ars_nouveau.glyph_desc.intangible": "使方块暂时变成空气。使用强化增强效果后，便可用于处理高挖掘等级的方块。它还能被范围效果、减少时效和延长时效增强。",
```

本来不太想改你的语序的，一个是太多第二个是还能接受，但是这句实在是太紧凑了，不拆不行。

之前轩辕说过，一段翻译出来的字段最好不要超过25个字，这样可以最大限度的保证阅读流畅感。你现在其实有很多都超了，不过改也没必要，多看一遍还是能看懂的。

向这种 for 后面跟一串目的的，可以考虑把它拆成单个分句，这样可以明显提升译文流畅度。在日常用语中，我们也会有“若想...那么...”或者“为了...因此”这种表因果的连接词，不要总是用“以”把目的和行为全缩在一起，中文有这样的表达，但是还是拆句子用的更多。